### DISCUSSION TITLE
Can I use Vectras VM to run Windows11 ARM version on Andorid?

### DISCUSSION THREAD

**Discussion by brandoncbd** [xoureldeen/Vectras-VM-Android#135] [Q&A]
Can I use Vectras VM to run Windows11 ARM version on Andorid? because there CPU simulation should NOT be necessary, and then performance is improved!
Furthermore, PC OS, such as Windows, is transiting to ARM more and more quickly, it is valuable to make Windows11 ARM version run properly, and even necessary to forget x86 PC OS and don't waste time on it.

**Comment by Windows-10X** [comment]
Hmm... Well. It technically IS possible but you will have to make it yourself. And if your device supports KVM that WILL speed it up. And also, you can set the CPU model as Host in the Create Command app by Nguyen Bao An Bui. Hope this helps

**Reply by Windows-10X** [reply]
Also, yes. I have looked myself
